### PR TITLE
download `last.state.txt` if DIFF_MODE=true

### PR DESCRIPTION
**Pull request by TomPohys**
Download `last.state.txt` file with `make download area=?` if `DIFF_MODE=true`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -289,6 +289,7 @@ ifeq (,$(wildcard $(PBF_FILE)))
 	@echo "Downloading $(DOWNLOAD_AREA) with replication support into $(PBF_FILE) and $(IMPOSM_CONFIG_FILE) from $(if $(OSM_SERVER),$(OSM_SERVER),any source)"
 	@$(DOCKER_COMPOSE) run $(DC_OPTS) openmaptiles-tools download-osm $(OSM_SERVER) "$(DOWNLOAD_AREA)" \
 				--imposm-cfg "$(IMPOSM_CONFIG_FILE)" \
+				--state /import/last.state.txt \
```
Comment (frodrigo) [inline Makefile:292]: The file should be named according to the area?

Comment (TomPohys) [reply Makefile:292]: so `--state /import/${DOWNLOAD_AREA}.state.txt` is better?

Comment (frodrigo) [reply Makefile:292]: Yes. But unsure for now from where and how the state file is used.

**Comment by frodrigo** [comment]
I made tests with geofabrik and osm-fr and it look like it is no necessary to download the state file. Imposm create it.

**Comment by TomPohys** [comment]
Could you please write a comment on how to create a state file by Imposm to #942 and #943? After that, I will close this PR. Thanks

**Comment by frodrigo** [comment]
I think we can close this one.

**Comment by TomPohys** [comment]
Thanks

**Pull request closed without merging**